setup.py

```diff
@@ -37,7 +37,7 @@
     'dill>=0.2.5, <0.3dev',
     'future>=0.16.0, <0.17dev',
     'googleapis-common-protos>=1.5.2, <2.0dev',
-    'grpcio>=1.0.0, <2.0dev',
+    'grpcio>=1.1.3, <2.0dev',
```

Comment:
I'd personally leave this at `1.0.0` or `1.1.0`, as this *works* on those version, you'll just have to build the wheel yourself.